### PR TITLE
special-interest-groups: add description for parser SIG

### PR DESCRIPTION
**Pull request by tangenta**

### CODE REVIEW

**Comment by gingerkidney** [comment]
lgtm

File: special-interest-groups/sig-parser/member-list.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+| [zhenghaoz](https://github.com/zhenghaoz) | 5              | 2019-08-12 11:19:59     | 2019-10-27 18:39:15   |
+
+## Contributors
+
+| user                                      | num_merged_PRs | earliest_PR_merged_time | latest_PR_merged_time |
+|:-----------------------------------------:|:--------------:|:-----------------------:|:---------------------:|
+| [hey-kong](https://github.com/hey-kong)   | 4              | 2019-09-04 23:32:46     | 2019-09-29 14:20:02   |
+| [huaouo](https://github.com/huaouo)       | 1              | 2019-08-15 14:28:09     | 2019-08-15 14:28:09   |
+
+##
```
Comment (zz-jason) [inline special-interest-groups/sig-parser/member-list.md]: what is this second head used for?

File: special-interest-groups/sig-parser/roles-and-organization-management.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+The minimal requirement for contributors to join the Parser SIG is to have
+the **Active Contributor** community role in the TiDB/parser repository.
+
+### Promotion
+
+#### From `Contributor` to `Active Contributor`
+
+Requirements:
+
+* Have at least 5 parser-related PRs merged within one year.
```
Comment (zz-jason) [inline special-interest-groups/sig-parser/roles-and-organization-management.md]: It should be 8, to be the same with our definition about `active contributors`

File: special-interest-groups/sig-parser/roles-and-organization-management.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+* Nominated by at least 2 Parser SIG Reviewers, Committer or Maintainer.
+
+#### From `Active Contributor` to `Reviewer`
+
+Requirements:
+
+* Have at least 15 parser-related PRs merged within one year.
```
Comment (zz-jason) [inline special-interest-groups/sig-parser/roles-and-organization-management.md]: how about 20?

File: special-interest-groups/sig-parser/member-list.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+## Active Contributors
+
+Members and their contributions:
+
+| user                                      | num_merged_PRs | earliest_PR_merged_time | latest_PR_merged_time |
+|:-----------------------------------------:|:--------------:|:-----------------------:|:---------------------:|
+| [spongedu](https://github.com/spongedu)   | 25             | 2018-11-26 12:50:20     | 2019-08-27 10:13:46   |
+| [leiysky](https://github.com/leiysky)     | 8              | 2019-07-30 11:07:34     | 2019-08-16 23:51:46   |
+| [lauhg](https://github.com/lauhg)         | 6              | 2019-08-13 12:47:51     | 2019-09-24 19:51:57   |
```
Comment (zz-jason) [inline special-interest-groups/sig-parser/member-list.md]: Form the definition of `active contributors`, they should have at least 8 PRs merged within 1 year. Seems we should update this list?

**Pull request merged**